pkg/cmd/openshift-operators/apis/operators/v1alpha1/types.go

```diff
@@ -0,0 +1,86 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type ManagementState string
+
+const (
+	// Enabled means that the operator is actively managing its resources and trying to keep the component active
+	Enabled ManagementState = "Enabled"
+	// Unmanaged means that the operator is not taking any action related to the component
+	Unmanaged ManagementState = "Unmanaged"
+	// Disabled means that the operator is actively managing its resources and trying to remove all traces of the component
+	Disabled ManagementState = "Disabled"
```

Comment:
so Disabled means the operator is trying to uninstall web console (iow. remove all created deployments/configs/etc.?)

Reply:
> so Disabled means the operator is trying to uninstall web console (iow. remove all created deployments/configs/etc.?)

correct.

Reply:
can we call it 'Remove' ?

Reply:
I think Unmanaged vs Disable is/will a source for confusion :)

Reply:
> I think Unmanaged vs Disable is/will a source for confusion :)

"Enabled" paired with "Remove"?   Can you make a set of three values with those meanings that you like?

Reply:
Managed, Unmanaged and Removed ?

Reply:
> anaged, Unmanaged and Removed ?

Updating.  We'll bikeshed later.